go.mod

```diff
@@ -4,11 +4,12 @@ go 1.12
 
 require (
 	github.com/containernetworking/cni v0.8.1
-	github.com/containers/common v0.35.2
-	github.com/containers/image/v5 v5.10.5
+	github.com/containers/common v0.34.2
+	github.com/containers/image/v5 v5.10.2-0.20210324124443-224346a8c9a7
```

Comment:
Go should know better ... it's really trying hard to match a commit to a version and fails so often :(

Looks like we need to wait for the next c/image release until we can merge.